get-covid-data/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
     mv mc /usr/bin/
 
 # COPY data    /data
-COPY src     /src
+# COPY src     /src
```

Comment:
Remove lines in favour of commenting them out.

Reply:
Leaving them in with an added comment to suggest that that's where a Dockerfile might get modified in another usecase